extlinks/organisations/templates/organisations/organisation_charts_include.html

```diff
@@ -166,8 +187,264 @@ <h3>Latest link events</h3>
     evt.currentTarget.className += " active";
   }
 
-  // Click all Link Events buttons
-  {% for collection_name, collection in collections.items %}
-    document.getElementById('{{ collection_name }}_linkEvents_button').click();
-  {% endfor %}
+  function getLinksCount(collection_id, form_data){
+    $.ajax({
+      url: "{% url 'organisations:links_count' %}?collection=" + collection_id + "&form_data=" + JSON.stringify(form_data),
+      beforeSend: function() {
+        document.getElementById(collection_id + "-links-added").innerHTML = "Loading...";
+        document.getElementById(collection_id + "-links-removed").innerHTML = "Loading...";
+        document.getElementById(collection_id + "-links-diff").innerHTML = "Loading...";
+      },
+      // on success
+      success: function(response) {
+        document.getElementById(collection_id + "-links-added").innerHTML = response.links_added;
+        document.getElementById(collection_id + "-links-removed").innerHTML = response.links_removed;
+
+        if (response.links_diff > 0) {
+         document.getElementById(collection_id + "-links-diff").innerHTML = "+" + response.links_diff;
+         document.getElementById(collection_id + "-links-diff").style.color = "green";
+       }
+       else{
+         document.getElementById("links-diff").innerHTML = response.links_diff;
```

Comment:
This line is throwing an exception in my dev console locally.

Reply:
What error is it throwing? I don't see any in mine.

Reply:
Just saw what it was! Good catch!

Reply:
```
9:562 Uncaught TypeError: Cannot set properties of null (setting 'innerHTML')
    at Object.success (9:562:58)
    at u (jquery.min.js:2:27457)
    at Object.fireWith [as resolveWith] (jquery.min.js:2:28202)
    at k (jquery.min.js:2:77651)
    at XMLHttpRequest.<anonymous> (jquery.min.js:2:79907)
```